projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/use-global-styles-config.js

```diff
@@ -12,13 +12,23 @@ export function useGlobalStylesConfig() {
 		const _globalStylesId = __experimentalGetCurrentGlobalStylesId
 			? __experimentalGetCurrentGlobalStylesId()
 			: null;
-		const globalStylesRecord = getEditedEntityRecord( 'root', 'globalStyles', _globalStylesId );
+
+		// Copy the global styles record to avoid mutating the original.
+		const globalStylesRecord = {
+			...getEditedEntityRecord( 'root', 'globalStyles', _globalStylesId ),
+		};
```

Comment:
Why do we need this? We’re only reading values, so the original record isn’t at risk of mutation. Also, using the spread operator `({ …obj })` creates only a shallow copy, any nested objects are still shared, so if you ever mutate them you’ll be touching the source.

Reply:
The config gets mutated in the second if of this PR and it affected the editor styles, so we needed a copy.